tests/Kernel/Paragraphs/ParagraphsTestBase.php

```diff
@@ -34,6 +34,9 @@ abstract class ParagraphsTestBase extends AbstractKernelTestBase {
     'oe_paragraphs',
     'allowed_formats',
     'typed_link',
+    'oe_paragraphs_timeline',
```

Comment:
Instead of adding extra dependencies (and adding more complexity to tests that don't need it), I think we should create a TimelineParagraphTest class and extend the dependencies there.